### PR TITLE
Remove console log statements 

### PR DESCRIPTION
**Pull request by MottMac-PH**
removed all console log statements from files in scope (i.e. not standard imported libraries managed by open source authors)

### CODE REVIEW

File: SiteAssets/pages/2.0/scripts/data.controller.1-GB020053AA.js [hunk 1]
```diff
@@ -151,7 +146,6 @@ tposdata = {
             })
             .fail(
                 function (error) {
-                    console.log(JSON.stringify(error));
```
Comment (JHolland98) [inline SiteAssets/pages/2.0/scripts/data.controller.1-GB020053AA.js:154]: We should keep this as it catches an error

File: SiteAssets/pages/2.0/scripts/data.controller.1.js [hunk 2]
```diff
@@ -150,7 +145,6 @@ tposdata = {
             })
             .fail(
                 function (error) {
-                    console.log(JSON.stringify(error));
```
Comment (JHolland98) [inline SiteAssets/pages/2.0/scripts/data.controller.1.js:153]: We should keep logging errors

File: SiteAssets/pages/3.0/scripts/CDM.DATA.CONTROLLER.js [hunk 3]
```diff
@@ -736,14 +710,12 @@ tposdata = {
...
             if(callback === "addHazards"){
                 toastr.success("hazard successfully added")
-                console.log("Success")
             }
         }
 
         function onFailure(sender, args){
             if(callback === "addHazards"){
                 toastr.error("hazard not added")
-                console.log(args.get_message() + '\n' + args.get_stackTrace());
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/CDM.DATA.CONTROLLER.js:746]: We should keep logging errors

File: SiteAssets/pages/3.0/scripts/CDM.DATA.CONTROLLER.js [hunk 4]
```diff
@@ -846,7 +818,6 @@ tposdata = {
                 $("#mainArea").append(ui.mkDataBox(itemsCount, "Hazards on system"));
             })
             .fail(function(error) {
-                //console.log(JSON.stringify(error));
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/CDM.DATA.CONTROLLER.js:849]: We should keep logging errors

File: SiteAssets/pages/3.0/scripts/DATA.FORMAT.CONTROLLER2.js [hunk 5]
```diff
@@ -265,15 +263,13 @@ formatdatato = {
...
                                     full_dataset = [...full_dataset, ...response];
-                                    //console.log("fairinput",flst);
                                     formatdatato.createTable(sublot_data, full_dataset, lst, trg ,flst, true)
                                     $(".loading-text").remove()
                                     // Allows for you to access the dashboard specific to your user roles. 
                                     activateDatasets(sublot_data, full_dataset);
                                 }
                             },
                             error: function(error) {
-                                //console.log(error);
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/DATA.FORMAT.CONTROLLER2.js:276]: We should keep logging errors

File: SiteAssets/pages/3.0/scripts/DATA.FORMAT.CONTROLLER2.js [hunk 6]
```diff
@@ -398,7 +392,6 @@ formatdatato = {
...
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/DATA.FORMAT.CONTROLLER2.js:401]: We should keep logging errors

File: SiteAssets/pages/3.0/scripts/DATA.FORMAT.CONTROLLER2.js [hunk 7]
```diff
@@ -813,7 +801,6 @@ formatdatato = {
                     formatdatato.hazardtablerows(response, next_url, number_loaded)
                 },
                 error: function(error) {
-                    //console.log(error);
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/DATA.FORMAT.CONTROLLER2.js:816]: We should keep logging errors

File: SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js [hunk 8]
```diff
@@ -3540,7 +3509,6 @@ function tposcustomfilters( data, forExport) {
                     timeOut: 30000
                 })
             }).catch((error) => {
-                console.log(error);
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js:3543]: We should keep logging errors

**Comment by MottMac-PH** [comment]
I've added back console.log(error) to all the places which had catch(error) or similar

**Review by JHolland98 — COMMENTED** [review]
There's a few places where you deleted error logs. Please can we keep these

**Review by JHolland98 — APPROVED** [review]

**Pull request merged**